docs/user/document/publisher/namespace-usage.en.md

```diff
@@ -1 +1,9 @@
 # Namespace - Usage
+
+The namespace is useful to unambiguously and globally identify the vendor or manufacturer.
+
+Often times a URL of the company or publishing party is used which is recommended.
+Can also be a URL pointing to a page on which advisories are published.
+
+In combination with the tracking ID (`document/tracking/id`) the CSAF document is uniquely identifiable.
+An additional version allows a version unique identification.
```

Comment:
That is correct (in principle). However, please make clear where that version comes from and what the result is.

```suggestion
If the document version is added to that combination, a single document is globally uniquely identified.